data/templates/ssh/sshd_config

```diff
@@ -9,13 +9,20 @@ ListenAddress 0.0.0.0
 Protocol 2
 # HostKeys for protocol version 2
 HostKey /etc/ssh/ssh_host_rsa_key
-HostKey /etc/ssh/ssh_host_dsa_key
+HostKey /etc/ssh/ssh_host_ed25519_key
+#HostKey /etc/ssh/ssh_host_dsa_key
```

Comment:
isn't this modification going to trigger a warning like "the key of this server has changed"?

Reply:
+1 ... we really need to test this carefully

Reply:
Could break some people using external automated tools to connect to their server (borg, nagios?, may be personnal cron)

Reply:
It was a bad idea to comments this lines. It's true sorry :+1: . This will generate a warning for an existing configuration.